src/ati/terraform.py

```diff
@@ -36,21 +37,25 @@ def iterresources(filenames):
     for filename in filenames:
         with open(filename, 'r') as json_file:
             state = json.load(json_file)
+            remote_state_func = ati.remote.get_remote_func(state)
+            if remote_state_func is not None:
+                state = remote_state_func(state)
```

Comment:
I was not (yet) trying to fix stuff from the source python file. Might come later.  I also need to expand testing first.